packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,386 +1,467 @@
+## 43.4.0 - June 21, 2018
```

Comment:
Format changes ('-' instead of '*') are being done by prettier. Let me know if that's a problem.

Reply:
I don't think that is a problem as long as everyone will be consistent with it moving forward.